avaframe/in3Utils/generateTopo.py

```diff
@@ -676,6 +676,11 @@ def generateTopo(cfg, avalancheDir):
     # If a drop shall be introduced
     if cfg['TOPO'].getboolean('drop'):
         z = addDrop(cfg, x, y, z)
+    
+    # moves topo in z direction 
+    if cfg['DEMDATA']['zEdit'] != '':
+    	z = z+cfg['DEMDATA'].getfloat('zEdit')
```

Comment:
Please make sure indentation is done via spaces and not tabs (lines 682 and 683) 

I recommend to change this in your editor (so spaces are used everytime you tab..)